en/identity-server/5.10.0/docs/references/permissions-required-to-invoke-admin-services.md

```diff
@@ -688,6 +688,12 @@ performed with different permission levels.
 </tbody>
 </table>
 
+!!! note "Important"
```

Comment:
Let's remove the topic inside the note. Instead, we can have the title in the note's title itself.

!!! note "Securing admin SOAP "services

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4603/commits/899ee1d073ce4c7d188ad21fdf20b24b51d22145